lib/appmap/command/agent_setup/status.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'appmap/service/config_analyzer'
+
+module AppMap
+  module Command
+    module AgentSetup
+      StatusStruct = Struct.new(:config_file)
+
+      class Status < StatusStruct
+        def perform
+          status = {
+            :properties => {
+              :config => {
+                :app => config_analyzer.app_name,
+                :present => config_analyzer.is_present?,
+                :valid => config_analyzer.is_valid?
+              },
+              :project => {
+                :agentVersionProject => AppMap::VERSION,
+                :language => 'ruby',
+                :remoteRecordingCapable => defined?(::Rails) ? true : false,
```

Comment:
I don't think that `defined?(::Rails)` will be reliable, because Rails might be in the bundle, but not loaded. Try looking for `Gem.loaded_specs['rails']` https://www.rubydoc.info/stdlib/rubygems/Gem.loaded_specs

Reply:
Of course we need to ensure that the code editor runs this script as `bundle exec <script-name>`, so that the whole bundle is present.